lib/game.rb

```diff
@@ -21,7 +21,7 @@ def initialize(game_data)
 
   def self.create_from_csv(file_path)
     games = []
-    CSV.foreach(file_path, headers: true, header_converters: :symbol) do |row|
+    CSV.foreach(file_path, headers: true, converters: :all) do |row|
```

Comment:
I like the converters: all but we'll have to make sure it doesn't change any of our data to a data type we're not expecting. Fine to keep for now though!